core/controllers/base_test.py

```diff
@@ -1206,6 +1210,14 @@ def test_controller_class_names(self):
                             class_return_type,
                             handler_type_to_name_endings_dict)
                     class_name = clazz.__name__
+                    # BulkEmailWebhookEndpoint is a unique class, compared to
+                    # others, since it is never called from the frontend, and so
```

Comment:
See above -- just because it's never called from the frontend doesn't mean someone can't just ping it directly, right?

Reply:
Added a decorator with authentication for it, though this comment just argues is just for the naming of the controller to not have 'Page' in it. A GET endpoint is required by the API to validate the URL and so it is added, it doesn't return anything (there is a comment in the get fn, explaining it).